src/sagas/task.saga.js

```diff
@@ -120,12 +113,16 @@ export function* launchDevServer({ task }: Action): Saga<void> {
             )
           );
           break;
+
         case 'exit':
           yield call(displayTaskComplete, task, message.wasSuccessful);
           yield put(
             completeTask(task, message.timestamp, message.wasSuccessful)
           );
           break;
+
+        default:
+          throw new Error('Unexpected channel for message: ' + message.channel);
```

Comment:
Admittedly this is a bit silly, but I figured if there _was_ a time where `message.channel` wasn't caught by one of the other branches, we probably want it to make some noise about it.

I think those are the only valid options? In my testing, I never saw this error.

Reply:
Can't hurt 😊